forms/contributor-signup.md

```diff
@@ -0,0 +1,40 @@
+Google form link: to be created
+
+# Mentored Sprint at PyCon US: Contributor Sign Up
+
+Please fill in this form if you would like to be mentored during the sprint.
+
+For more information, please check: https://us.pycon.org/2019/hatchery/mentoredsprints/
+
+1. Email address (required)
+
+2. Your name (required)
+
+3. Are you a member of underrepresented group?
+
+4. Preferred open source projects to contribute to (should this be required?)
+
+5. Topic Interest areas:
+   
+   E.g: web, data science, machine learning, frontend, etc
+
+6. What kind of contribution are you interested in doing?
+   
+   E.g: code (which language), documentation, UI design, etc.
+
+7. Availability during the sprint (required)
+
+   [ ] Two hours
+
+   [ ] Four hours
+
+8. Have you attended a sprint or hackathon before? If yes, which one(s)?
+
+9. What is your level of familiarity with Git?
+
+   Any skill level is welcome! We plan to have a git helpdesk.
+
+10. Do you have your own laptop for the sprint, or will you require a loaner?
```

Comment:
Not yet sure if this is something we can accommodate, but figured it would be good to ask?

Reply:
I would change this message a little bit, just in case we can't have loaners.

Maybe something like
> Will you be able to bring your own laptop to the sprint? If not, there is a small chance that we will have extra computers available for your use.

Is that too verbose?

Reply:
I like this suggestion even if verbose :)